src/explore-education-statistics-common/src/modules/table-tool/components/PublicationForm.tsx

```diff
@@ -78,142 +115,131 @@ const PublicationForm = ({
-                            <DetailsMenu
-                              summary={topic.title}
-                              key={topic.id}
-                              id={`${formId}-topic-${topic.id}`}
-                              detailsId="topic"
-                              open={
-                                searchTerm !== '' ||
-                                topic.publications.some(
-                                  publication =>
-                                    publication.id === values.publicationId,
-                                )
-                              }
-                            >
-                              <FormFieldRadioGroup
-                                legend={`Choose option from ${topic.title}`}
-                                legendHidden
-                                small
-                                showError={false}
-                                name="publicationId"
-                                disabled={form.isSubmitting}
-                                options={topic.publications.map(
-                                  publication => ({
-                                    label: publication.title,
-                                    value: publication.id,
-                                  }),
-                                )}
-                              />
-                            </DetailsMenu>
-                          ))}
-                        </DetailsMenu>
-                      ))
-                    ) : (
-                      <p>No publications found</p>
+                <p>or</p>
+
+                <div className={styles.optionsContainer}>
+                  <FormRadioGroup
+                    id={`${formId}-themes`}
+                    legend="Select a theme"
+                    legendSize="s"
+                    name="themeId"
+                    small
+                    options={themes.map(theme => {
+                      return {
+                        label: theme.title,
+                        value: theme.id,
+                      };
+                    })}
+                    value={selectedThemeId}
+                    onChange={e => {
+                      setSelectedThemeId(e.target.value);
+                      form.setFieldValue('publicationId', '');
+                      setSearchTerm('');
+                    }}
+                  />
+
+                  <div className={styles.publicationsList} id="publications">
+                    <FormFieldRadioGroup
+                      id={`${formId}-publications`}
+                      legend={
+                        <>
+                          Select a publication
+                          <span
+                            className="govuk-visually-hidden"
+                            aria-live="polite"
+                            aria-atomic="true"
+                          >
+                            {' '}
```

Comment:
This space can just be added to the front of the adjacent template string to avoid splitting the node up:

```ts
` ${publications.length} ...`
```